spacy/lang/fr/stop_words.py

```diff
@@ -36,45 +37,48 @@
 l' l’ la laisser laquelle le lequel les lesquelles lesquels leur leurs longtemps
 lors lorsque lui lui-meme lui-même là lès
 
-m' m’ ma maint maintenant mais malgre me meme memes merci mes mien
+m' m’ ma maint maintenant mais malgre malgré me meme memes merci mes mien
 mienne miennes miens mille moi moi-meme moi-même moindres moins
 mon même mêmes
 
 n' n’ na ne neanmoins neuvième ni nombreuses nombreux nos notamment
-notre nous nous-mêmes nouvea nul néanmoins nôtre nôtres
+notre nous nous-mêmes nouveau nul néanmoins nôtre nôtres
 
-o ô on ont onze onzième ore ou ouias oust outre
+o ô on ont onze onzième or ou ouias ouste outre
 ouvert ouverte ouverts où
 
-par parce parfois parle parlent parler parmi parseme partant
+par parce parfois parle parlent parler parmi partant
 pas pendant pense permet personne peu peut peuvent peux plus
-plusieurs plutôt possible possibles pour pourquoi
-pourrais pourrait pouvait prealable precisement premier première premièrement
-pres procedant proche près pu puis puisque
+plusieurs plutot plutôt possible possibles pour pourquoi
+pourrais pourrait pouvait prealable precisement
+premier première premièrement
+pres procedant proche près préalable précisement pu puis puisque
 
-qu' qu’ quand quant quant-à-soi quanta quarante quatorze quatre quatre-vingt
+qu' qu’ quand quant quant-à-soi quarante quatorze quatre quatre-vingt
 quatrième quatrièmement que quel quelconque quelle quelles quelqu'un quelque
 quelques quels qui quiconque quinze quoi quoique
 
 relative relativement rend rendre restant reste
-restent retour revoici revoilà
+restent retour revoici revoila revoilà
 
 s' s’ sa sait sans sauf se seize selon semblable semblaient
 semble semblent sent sept septième sera seraient serait seront ses seul seule
-seulement si sien sienne siennes siens sinon six sixième soi soi-même soit
-soixante son sont sous souvent specifique specifiques stop
+seulement seuls seules si sien sienne siennes siens sinon six sixième soi soi-meme soi-même soit
+soixante son sont sous souvent specifique specifiques spécifique spécifiques stop
```

Comment:
- added "seuls" and "seules" (plural forms);
- added "spécifique" "spécifiques" (correct accented forms);